test/e2e/e2e_test.go

```diff
@@ -103,6 +105,26 @@ func TestMain(m *testing.M) {
 		AssetNames: generated.AssetNames,
 	})
 
+	if framework.TestContext.ListConformanceTests {
+		var tests []struct {
```

Comment:
Ideally this struct should be imported from a file in the conformance directory to prevent a yaml structure change from breaking this. Right now this seems to be the best option since the conformance `walk.go` file is in a main package.

Reply:
thanks @Jefftree